apps/settings/src/components/PersonalInfo/ProfileSection/EditProfileAnchorLink.vue

```diff
@@ -58,7 +58,7 @@ a {
 	line-height: 44px;
 	padding: 0 16px;
```

Comment:
Similar here, the padding looks weird with a that small border radius.

```suggestion
	padding: 0 calc(var(--default-grid-baseline) * 2);
```